src/GroupManager.php

```diff
@@ -93,6 +102,11 @@ public function addGroup($entity_type_id, $bundle_id) {
     $editable->set('groups', $groups);
     $saved = $editable->save();
 
+    // Notify other module we added a new group.
+    // todo: should this be an event?
+    $this->attachUserField($entity_type_id, $bundle_id);
```

Comment:
@pfrenssen I tried to keep the logic in here but because the container is not set with the proper mangers(entity manager, field manager) the test is going crazy. Even when trying to set the container manually as we do in other tests it's not working because we trying to load fields from the DB.

I'll think i'll revert and we can try in the future to put the logic in the group manager.